semantic_conventions/resource/telemetry.yaml

```
@@ -10,6 +10,13 @@ groups:
         requirement_level: required
         brief: >
           The name of the telemetry SDK as defined above.
+        note: |
+          The OpenTelemetry SDK MUST set the `telemetry.sdk.name` attribute to `opentelemetry`.
```

Another alternative would be:

```suggestion
          SDKs provided by the OpenTelemetry project MUST set the `telemetry.sdk.name` attribute to `opentelemetry`.
```

To emphasize that "oficial" OTel SDKs must do that.

---

We can do it in a separate PR (similar comment https://github.com/open-telemetry/opentelemetry-specification/pull/3392#discussion_r1166961838)